### PR TITLE
Feat/hero card

### PR DESCRIPTION
**Pull request by Vikingjonsson**

### CODE REVIEW

**Comment by Vikingjonsson** [comment]
Misguiding branch name, the component is now called SelectionCard. This aims to replace "GuidanceCard" and "IconCard".

**Review by victoriatomzik — APPROVED** [review]

File: src/Molecules/SelectionCard/SelectionCard.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import { action } from '@storybook/addon-actions';
+
+import { Playground } from '../../../docs/Playground';
+import { SelectionCard } from './SelectionCard';
+
+# Multi Step Progress
```
Comment (Vikingjonsson) [inline src/Molecules/SelectionCard/SelectionCard.mdx:7]: will fix

**Review by stigdanielek — APPROVED** [review]

**Review by crewneck — APPROVED** [review]

**Review by LifeofMichal — APPROVED** [review]

**Review by mjmonline — CHANGES_REQUESTED** [review]
I can't reach the component with keyboard or screen reader. Also the component is controlled only, would be nice to have the option to run it with uncontrolled behavior

**Review by kmelkon — APPROVED** [review]

**Review by mjmonline — APPROVED** [review]
👍

**Pull request merged**